traefik/templates/_podtemplate.tpl

```diff
@@ -150,7 +150,7 @@
           {{- end }}
           {{- range $name, $config := .Values.ports }}
           {{- if $config }}
-          - "--entrypoints.{{$name}}.address=:{{ $config.port }}/{{ default "tcp" $config.protocol | lower }}"
+          - "--entrypoints.{{$name}}.address={{ $config.host }}:{{ $config.port }}/{{ default "tcp" $config.protocol | lower }}"
```

Comment:
```suggestion
          - "--entrypoints.{{$name}}.address={{ $config.hostIP }}:{{ $config.port }}/{{ default "tcp" $config.protocol | lower }}"
```

Wdyt about using already existing `hostIP` instead of introducing another setting for the same purpose ?

Reply:
I have tested it (setting `hostIP` and `host` in arguments to the same value) and see no issues. I think it's a right thing to do.